modules/restarting-installation.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// installing/installing-troubleshooting.adoc
+
+[id="restarting-installation_{context}"]
+= Reinstalling the {product-title} cluster
+
+If you are unable to debug and resolve issues in the failed {product-title} installation, you might need to consider installing a new {product-title} cluster. Before starting the installation process again, you must do a thorough cleanup.
```

Comment:
```suggestion
If you are unable to debug and resolve issues in a failed {product-title} installation, consider installing a new {product-title} cluster. Before starting the installation process again, you must complete thorough cleanup.
```

Changes suggested for more minimalistic and direct writing.

Reply:
Addressed.